_config.yml

```diff
@@ -99,3 +99,4 @@ format:
     periods: '<a href="/docs/1.0/api/#cash_flow_overtime_periods" alt=""/>Periods</a>'
     top_customers: 'array[<a href="/docs/1.0/api/#top_customer" alt=""/>Top customer</a>]'
     top_invoiced: 'array[<a href="/docs/1.0/api/#top_invoiced" alt=""/>Top invoiced</a>]'
+    snippet_customer: '<a href="/docs/1.0/api/#snippet" title="">Snippet</a>&lt;<a href="/docs/1.0/api/#customer" title="">Customer</a>&gt;'
```

Comment:
Actually, that type should be more generic and just 'snippet'.

We will have more snippet in the futur, like Invoice snippet, etc... they should all be linked to the 'snippet' description of the documentation.